packages/web-service/src/pages/habitat/name/habitat-name.js

```diff
@@ -1,6 +1,15 @@
+import Joi from 'joi'
 import pageRoute from '../../../routes/page-route.js'
 import { habitatURIs } from '../../../uris.js'
+const nameReg = /^[/\s\p{L}-]{1,160}$/u
```

Comment:
On the page this is limited to 100 characters so probably best to align this with the input length.
I think we are almost certainly going to want to allow numerics and -,._ so please experiment with ^[/\s0-9\._,\p{L}-]{1,100}$.